### ISSUE TITLE
Increase DELAY more 50000ms, stops the WSGI server

### ISSUE THREAD

**Issue by Milindsoni08** [dbisu/pico-ducky#205]
Increase DELAY more 50000ms, stops the WSGI server and i am unable to access 192.168.4.1
payload.dd
DELAY 50000

if i keep DELAY < 50000ms it works like a charm

**Comment by Karthiksgowda** [comment]
The above mentioned solution is not working @dbisu

**Comment by dbisu** [comment]
Yeah, I'm still working on a solution. Like I said, might take a bit to figure out the right way to refactor the code.

**Comment by ihsaank** [comment]
Hi, I'm only able to connect to the pico w for literally 30 seconds then the web server stops loading. Unless I forget the SSID and reconnect then it only works for a few seconds again. Do you know what the issue is?

**Comment by dbisu** [comment]
> Hi, I'm only able to connect to the pico w for literally 30 seconds then the web server stops loading. Unless I forget the SSID and reconnect then it only works for a few seconds again. Do you know what the issue is?

@ihsaank please open a separate issue for this problem.

**Comment by dbisu** [comment]
For the original problem, I've created a test branch, delay-refactor, with proposed changes to fix this issue.

**Comment by ihsaank** [comment]
Is the original issue the same as the problem I’ve stated here?

**Comment by dbisu** [comment]
Are you trying to use long delays and seeing the webserver stop?
A log dump from the debug serial would be helpful.

**Comment by ihsaank** [comment]
Oh okay sorry no it’s a different issue I’m facing. I’ve followed all the installation steps correctly so not sure if it’s a firmware issue. Once I’m connected to the SSID I’m only able to execute payloads only for a few seconds then the web server freezes up and stops loading and seems to show I can no longer connect and when I go to the wifi option it shows a cross across the wifi and ‘no internet connection’ underneath it

**Comment by dbisu** [comment]
This is a different problem, please open a separate issue to track it.
